service/static/js-src/muk.institution.js

```diff
@@ -3571,6 +3573,13 @@ $.extend(edges, {
             }
         };
         this.synchronise = function() {
+            if(initialPageLoad) {
+                initialPageLoad = false
+                var institutionList = this.edge.result.buckets("institution");
+                for (var i = 0; i < 9; i++) {
```

Comment:
Why '9' ? how do you know there will always be at least  9 instititutions? What if more / fewer?